docs/cudf/source/user_guide/guide-to-udfs.ipynb

```diff
@@ -7,13 +7,24 @@
     "# Overview of User Defined Functions with cuDF"
```

Comment:
<p>&gt;we just need to pre-allocate an output array and leverage the <code>forall</code> method mentioned above</p><p> </p><p>I can't remember where I first learned about avoiding the 'J' word <em>just</em> - but it was probably [here](https://software-carpentry.org/blog/2015/03/teaching-tips.html).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/10607/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>